packages/dapp/src/embeds/StoryBoostLoader.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from "react";
+import styled, { ThemeProvider } from "styled-components";
+import { useRouteMatch } from "react-router";
+import {
+  CivilContext,
+  ICivilContext,
+  RENDER_CONTEXT,
+  CivilIcon,
+  DEFAULT_BUTTON_THEME,
+  DEFAULT_CHECKBOX_THEME,
+} from "@joincivil/components";
+import { mediaQueries } from "@joincivil/elements";
+import { StoryBoost } from "@joincivil/sdk";
+
+import { embedRoutes } from "../constants";
+import AppProvider from "../components/providers/AppProvider";
+
+const EmbedWrapper = styled.div`
+  background: white; // obscure embed loading message outside iframe
+`;
+const CivilLogoLink = styled.a`
+  position: absolute;
+  display: inline-block;
+  z-index: 2; // above basic stuff, below full screen modal mask
+  top: 0;
+  right: 0;
+  padding: 20px 30px 0 20px;
+  background: rgb(255, 255, 255);
+  background: linear-gradient(90deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 1) 35%);
+  ${mediaQueries.MOBILE} {
+    padding-right: 10px;
+  }
+  ${mediaQueries.MOBILE_SMALL} {
+    padding-top: 22px;
+    padding-left: 60px;
+    background: linear-gradient(90deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 1) 40%);
+  }
+  svg {
+    width: 50px;
+    height: auto;
+  }
+`;
+
+export interface StoryBoostLoaderParams {
+  boostId: string;
+}
+
+const StoryBoostLoaderComponent: React.FunctionComponent = () => {
+  const civilContext = React.useContext<ICivilContext>(CivilContext);
+  civilContext.renderContext = RENDER_CONTEXT.EMBED;
+  const theme = {
+    ...DEFAULT_CHECKBOX_THEME,
+    ...DEFAULT_BUTTON_THEME,
+    renderContext: RENDER_CONTEXT.EMBED,
+  };
+
+  // Due to a conflict between react-router v5's `BrowserRouter`, which we use, and react/redux `ConnectedRouter`, which we also use (and which would take an unknown/large refactor to change without breaking code splitting gains), neither `useParams` hook nor `withRouter` are receiving updates here, so we have to use `useRouteMatch` and manually provide the boost embed route. - @tobek
```

Comment:
is there a Jira to track this? probably should put it on our radar to fix eventually

Reply:
There is! https://civilmedia.atlassian.net/browse/CIVIL-1121

That comment is copy-pasted from previous `BoostLoader.tsx` - the two files could be DRY'd up a lot, but I'm not yet sure what all actually needs re-using as things are still a bit in flux as I navigate implementation issues.